clients/fides-js/src/components/fides.css

```diff
@@ -1063,16 +1070,6 @@ div#fides-overlay-wrapper .fides-i18n-pseudo-button {
   transform: rotate(180deg);
 }
 
-.fides-i18n-menu:hover .fides-i18n-popover,
-.fides-i18n-menu:focus-within .fides-i18n-popover {
```

Comment:
hover / focus is now react-driven. These default CSS properties (when popover is shown) has been moved up to the `.fides-i18n-popover` selector